src/screens/Events/Events.js

```diff
@@ -1,82 +1,63 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { Alert, Button, ButtonLayout, Agenda } from '@/components/';
-import { formatEventListForCalendar, prepend0 } from '@/utils/events';
-import { loadEvent, createEvent } from '@/ducks';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Button, ButtonLayout } from '@/components/';
+import { formatEventListForCalendar, prepend0, DefaultAgenda } from '@/utils/events';
+import { createEvent } from '@/ducks';
 import { EventForm } from '@/data/FormData';
 import { View, Dimensions, SafeAreaView } from 'react-native';
 
-const dimension = Dimensions.get('screen');
+const { height } = Dimensions.get('screen');
 
-class Events extends Component {
-	constructor(props) {
-		super(props);
+const getTodaysDate = () => {
+	let date = new Date();
+	let month = prepend0((date.getMonth() + 1).toString());
+	let year = date.getFullYear();
+	let day = prepend0(date.getDate().toString());
```

Comment:
Maybe we should even make this a function...
```js
const getDate(format) => [day, month, year]
/**
  * param: format ('number' || 'string')
  * returns array of [day, month, year] as
  * -> numbers: [1, 1, 2021]
  * -> words:   ['January', '1', '2021']
*/
```

---

```suggestion
	let date = new Date();
	let month = (date.getMonth() + 1).toString().padStart(2, '0');
	let year = date.getFullYear();
	let day = date.getDate().toString().padStart(2, '0');
```

Reply:
Yea we can probably make that in one up of the upcoming PRs